net.peercoin.peercoin-qt.json

```diff
@@ -0,0 +1,73 @@
+{
+  "id": "net.peercoin.peercoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "20.08",
+  "command": "peercoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "peercoin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/peercoin-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/peercoin/peercoin/releases/download/v0.10.4ppc/peercoin-0.10.4-x86_64-linux-gnu.tar.gz",
+          "sha256": "59af6c2a3102da91d8dfc6d1bb5da1c5f21825bb878f6741518b3eeaae51c361"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["aarch64"],
+          "url": "https://github.com/peercoin/peercoin/releases/download/v0.10.4ppc/peercoin-0.10.4-aarch64-linux-gnu.tar.gz",
```

Comment:
It's not a blocker, but please consider adding https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

Reply:
I will look into that.